packages/create-workleap/index.ts

```diff
@@ -0,0 +1,109 @@
+import * as child_process from "child_process";
+import process from "process";
+
+import { Intro, Select, Note, Text, Outro } from "./prompts";
+import * as pkg from "./package.json";
+import { StartSpinner, StopSpinner } from "./spinner";
+
+const DEFAULT_OUTPUT_DIRECTORY = "./my-new-project";
+const NAME_PARAMETER_POSITION = 2; // TODO validate position of parameter once ask with `pnpm create`
```

Comment:
Only used 1 time, should it really be extracted?

If it's used more than one time, it seems like a good opportunity to add an inline utility function `getNameArgument`